src/js/themes/darkTheme/colors.ts

```diff
@@ -54,31 +56,37 @@ export const colors: ThemeColors = {
   },
   error: {
     default: '#D73847',
-    alternative: '#E06470',
+    defaultHover: '#FF4D58',
```

Comment:
Matches figma and has css in js and css variable token

![Screenshot 2024-01-18 at 5 58 57 PM](https://github.com/MetaMask/design-tokens/assets/8112138/1b78d0b5-ede5-436d-bea1-5982801d0f1e)